platform/platform_tests/distance_tests.cpp

```diff
@@ -256,21 +265,21 @@ UNIT_TEST(Distance_FormattedDistance)
     {Distance(991,     Units::Meters),     990,  Units::Meters,     "990",  "990 m"},
 
     // From Kilometers to Kilometers
-    {Distance(0,       Units::Kilometers), 0,    Units::Meters,     "0",    "0 m"},
-    {Distance(0.3,     Units::Kilometers), 300,  Units::Meters,     "300",  "300 m"},
-    {Distance(1.234,   Units::Kilometers), 1.2,  Units::Kilometers, "1.2",  "1.2 km"},
-    {Distance(10,      Units::Kilometers), 10,   Units::Kilometers, "10",   "10 km"},
-    {Distance(11,      Units::Kilometers), 11,   Units::Kilometers, "11",   "11 km"},
-    {Distance(54,      Units::Kilometers), 54,   Units::Kilometers, "54",   "54 km"},
-    {Distance(99.99,   Units::Kilometers), 100,  Units::Kilometers, "100",  "100 km"},
-    {Distance(100.01,  Units::Kilometers), 100,  Units::Kilometers, "100",  "100 km"},
-    {Distance(115,     Units::Kilometers), 115,  Units::Kilometers, "115",  "115 km"},
-    {Distance(999,     Units::Kilometers), 999,  Units::Kilometers, "999",  "999 km"},
-    {Distance(1000,    Units::Kilometers), 1000, Units::Kilometers, "1000", "1000 km"},
-    {Distance(1049.99, Units::Kilometers), 1050, Units::Kilometers, "1050", "1050 km"},
-    {Distance(1050,    Units::Kilometers), 1050, Units::Kilometers, "1050", "1050 km"},
-    {Distance(1050.01, Units::Kilometers), 1050, Units::Kilometers, "1050", "1050 km"},
-    {Distance(1234,    Units::Kilometers), 1234, Units::Kilometers, "1234", "1234 km"},
+    {Distance(0,       Units::Kilometers), 0,    Units::Meters,     "0",     "0 m"},
+    {Distance(0.3,     Units::Kilometers), 300,  Units::Meters,     "300",   "300 m"},
+    {Distance(1.234,   Units::Kilometers), 1.2,  Units::Kilometers, "1.2",   "1.2 km"},
+    {Distance(10,      Units::Kilometers), 10,   Units::Kilometers, "10",    "10 km"},
+    {Distance(11,      Units::Kilometers), 11,   Units::Kilometers, "11",    "11 km"},
+    {Distance(54,      Units::Kilometers), 54,   Units::Kilometers, "54",    "54 km"},
+    {Distance(99.99,   Units::Kilometers), 100,  Units::Kilometers, "100",   "100 km"},
+    {Distance(100.01,  Units::Kilometers), 100,  Units::Kilometers, "100",   "100 km"},
+    {Distance(115,     Units::Kilometers), 115,  Units::Kilometers, "115",   "115 km"},
+    {Distance(999,     Units::Kilometers), 999,  Units::Kilometers, "999",   "999 km"},
+    {Distance(1000,    Units::Kilometers), 1000, Units::Kilometers, "1,000", "1,000 km"},
```

Comment:
If we change defaults to >4 chars for group separators, these tests will change, right?

Reply:
Yes, the UTs have to be changed.